python/main_files/gui-project/gui.py

```diff
@@ -133,48 +173,77 @@ def set_pitch(self,pitch):
             self.setting_pitch_error("Pitch is not an int")
 
     def setting_pitch_error(self,error_message):
-            print(f"\a{error_message}")
-            self.__entry_box.delete(0,END)
-            self.__entry_box.insert(0,self.get_pitch())
+        """Printing passed error message and clearing entry box"""
+        print(f"\a{error_message}")
+        self.__entry_box.delete(0,END)
+        self.__entry_box.insert(0,self.get_pitch())
 
     def get_pitch(self):
+        """Getter for pitch"""
         return self.__pitch_value
 
     def send_note_on(self):
+        """
+        Core function to send note_on message to the midi device.
+        It will send note_on message with the current pitch and velocity,
+        Change the label under the info,
+        Send control_change message (?)
```

Comment:
I'm not sure about sending control_change message every time sending note_on message. I'll test not sending this message when I'm available.